addons/cookoff/script_component.hpp

```diff
@@ -24,6 +24,8 @@
 #define SMOKE_TIME 10.5
 #define COOKOFF_TIME 14 // Cook off time should be 20s at most due to length of sound files
 #define COOKOFF_TIME_BOX 82.5 // Cook off time for boxes should be significant to allow time for ammo to burn
+#define MIN_AMMO_DETONATION_START_DELAY 1 // Min time to wait before a vehicle's ammo starts to cookoff
+#define MAX_AMMO_DETONATION_START_DELAY 6 // Max time to wait before a vehicle's ammo starts to cookoff
```

Comment:
Could use a blank line to separate these from the macros above since the comment above those does not apply to them